src/arrayMethodIndexOf.js

```diff
@@ -5,8 +5,26 @@
  *
  */
 function applyCustomIndexOf() {
-  [].__proto__.indexOf2 = function(searchElement, fromIndex) {
-    // write code here
+  [].__proto__.indexOf2 = function(searchElement, fromIndex = 0) {
+    if (fromIndex >= this.length) {
+      return -1;
+    }
+    let index = 0;
+    fromIndex < 0 ? index = 0 : index = fromIndex;
+    for (index; index < this.length; index++) {
+      if (typeof searchElement === typeof this[index]) {
+        if (searchElement === this[index]) {
+          return index;
+        } else if (isNaN(searchElement) && isNaN(this[index])) {
+          return index;
+        } else {
+          continue;
```

Comment:
do not use continue
https://github.com/mate-academy/js_array-method-lastIndexOf/pull/8#discussion_r275220303